dandi/bids_validator_xs.py

```diff
@@ -450,7 +452,7 @@ def validate_all(
 
 def write_report(
     validation_result,
-    report_path="/var/tmp/bids-validator-report_{}.log",
+    report_path="{logdir}/bids-validator-report_{date}.log",
     datetime_format="%Y%m%d-%H%M%S",
```

Comment:
```suggestion
    datetime_format="%Y%m%d%H%M%SZ",
```
per above comment, and use utcnow